maintainers/maintainer-list.nix

```diff
@@ -8963,6 +8963,16 @@
     githubId = 59476;
     name = "Peter Schuller";
   };
+  scoder12 = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.